extras/help_generator/generate_help.py

```diff
@@ -131,5 +127,3 @@
 
             all_data = coll_data(keywords, documentation, num, helpdir, fname,
                                  sli_command_list)
-
-write_helpindex(helpdir)
```

Comment:
I disagree with this change. IMO `make` should *create* all files, while `make install` just *installs* them and, if necessary, takes care that the installation directory is updated (e.g. by running `generate_helpindex.py`again).

Either this functionality is re-added  to this script or an additional call to `generate_helpindex.py` has to be added to `CMakeLists.txt`which calls  separately. I prefer the first option for its simplicity.